pkg/autoscaler/autoscaler/autoscaler_manager.go

```diff
@@ -151,6 +151,9 @@ func (am *autoScalerManager) updateAutoScaling(oldTc *v1alpha1.TidbCluster,
 		return nil, nil
 	}
 
+	// record autoscaler sync timestamp
+	tac.Annotations[label.AnnStsLastSyncTimestamp] = fmt.Sprintf("%d", time.Now().Unix())
```

Comment:
@vincent178 I think AnnStsLastSyncTimestamp is not suitable in this case as this annotation is used to resync the statefulset in periodicity controller. What about using a new annotation named as AnnLastTikvAutoScaleingSyncTimestamp